domains/queaxtra.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Queaxtra",
+        "email": "fatihbgr@duck.com"
+    },
+    "record": {
+        "TXT": "vc-domain-verify=queaxtra.is-a.dev,99c8c889ab11cf641590"
```

Comment:
Vercel is not supported

Reply:
Can you please make the record value "CNAME" and "https://queaxtra-me.pages.dev/"? I migrated the site to Cloudflare Pages.

Reply:
You can do it yourself, just edit the file in your fork